src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -0,0 +1,36 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.article.Article;
+import kr.codesqaud.cafe.article.MemoryArticleRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class ArticleController {
+
+    private final MemoryArticleRepository memoryArticleRepository;
+    private final Logger logger = LoggerFactory.getLogger(UserController.class);
```

Comment:
```suggestion
    private final Logger logger = LoggerFactory.getLogger(ArticleController.class);
```
현재 클래스 메타데이터를 가져오는 Reflection Method 사용도 고려해보세요~